api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -127,6 +127,12 @@ func (in *DatasourceList) DeepCopyObject() runtime.Object {
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
 func (in *DatasourceSpec) DeepCopyInto(out *DatasourceSpec) {
 	*out = *in
+	in.Enpoint.DeepCopyInto(&out.Enpoint)
+	if in.OSS != nil {
+		in, out := &in.OSS, &out.OSS
+		*out = new(OSS)
+		**out = **in
+	}
```

Comment:
**Suggestion:** It seems like you are manually adding DeepCopyInto() methods for your custom types (e.g., DatasourceSpec, EmbedderSpec, etc.). This is not necessary as the deepcopy-gen tool can generate these methods for you. You can use the "// +k8s:deepcopy-gen=true" annotation in your type definitions to instruct deepcopy-gen to generate the DeepCopyInto() method.
```suggestion
// No need to manually write this method, it can be auto-generated by deepcopy-gen tool.
```